cmd/admission-webhook-server/main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/golang/glog"
+
+	controller "github.com/kinvolk/lokomotive/internal/admission-webhook-server"
+)
+
+const (
+	port = "8080"
+)
+
+func usage() {
+	flag.PrintDefaults()
+	os.Exit(0)
+}
+
+func returnError(msg string, err error) {
+	glog.Fatalf("%s: %v", msg, err)
+}
+
+func main() {
+	var tlscert, tlskey, logtostderr, stderrthreshold, loglevel string
+
+	flag.Usage = usage
+
+	flag.StringVar(&tlscert, "tlsCertFile", "/etc/certs/cert.pem", "File containing the x509 Certificate for HTTPS.")
+	flag.StringVar(&tlskey, "tlsKeyFile", "/etc/certs/key.pem", "File containing the x509 private key to --tlsCertFile.")
+	flag.StringVar(&logtostderr, "logToStderr", "true", "Logs are written to files instead of to standard error.")
+	flag.StringVar(&stderrthreshold, "stdErrThreshold", "WARNING", "Severity level for standard error.")
+	flag.StringVar(&loglevel, "loglevel", "2", "Enable V-leveled logging at the specified level.")
+	flag.Parse()
+
+	err := flag.Set("logtostderr", logtostderr)
```

Comment:
If we have defaults in `flag.StringVar`, we no longer need to set?

Reply:
We cannot use the same flag `logtostderr`, because it returns error `flag redefined: logtostderr`. That is why setting it seperately